core/Plugin.php

```diff
@@ -451,6 +451,19 @@ public static function getPluginNameFromNamespace($namespaceOrClassName)
     }
 
     /**
+     * Override this method in your plugin class if you want your plugin to be loaded during tracking.
+     *
+     * Note: If you define your own dimension or handle a tracker event, your plugin will automatically
+     * be detected as a tracker plugin.
+     *
+     * @return bool
+     */
+    public function isTrackerPlugin()
```

Comment:
I'd prefer to have only one to keep things simple and to have good examples everywhere. If it's done differently everywhere it's not clear for users how to solve it. I created a trello card for our meetup next week so we can discuss. As request processors are not API it's not so important anyway